src/redux/action-creators/registration.action-creator.js

```diff
@@ -0,0 +1,61 @@
+import {
+  success,
+  failure,
+  requested,
+  InputChange,
+  validateInput
+} from "../actions/registration.action";
+import registerUserService from "../../services/registration.service";
+import validator from "../../utils/validator";
+
+const handleRegistration = user => dispatch => {
+  dispatch(requested(user));
+  return registerUserService(user).then(
```

Comment:
either use try and catch, or then and catch.

Reply:
I'm using then and catch here...